README.md

```diff
@@ -221,6 +222,34 @@ segment is downloaded. You can override this function to provide your
 adaptive streaming logic. You must, however, be sure to return a valid
 media playlist object that is present in `player.hls.master`.
 
+Overridding this function with your own is very powerful but is overkill
+for many purposes. Most of the time, you should use the much simpler
+function below to selectively enable or disable a playlist from the
+adaptive streaming logic.
+
+#### hls.representations
+Type: `function`
+
+To get all of the available representations, call the `representations()` method on `player.hls`. This will return a list of plain objects, each with `width`, `height`, `bandwidth`, and `id` properties, and an `enabled()` method.
+
+```javascript
+player.hls.representations();
+```
+
+To see whether the representation is enabled or disabled, call its `enabled()` method with no arguments. To set whether it is enabled/disabled, call its `enabled()` method and pass in a boolean value.
```

Comment:
can we call out explicitly that `enabled(true)` enables the representation and that `enabled(false)` disabled it?